### ISSUE TITLE
Trouble with domain added with manual name servers

### ISSUE THREAD

**Issue by sethtjf** [aws/copilot-cli#4172]
Hey Copilot team,

I am trying to create an app with the --domain flag. This is a domain that was not registered through Route 53. I followed the guides to add the correct name servers to a hosted zone in Route53. However, I still get the below error message:

```
Note: The account does not seem to own the domain that you entered.
Please make sure that MY_DOMAIN_NAME is registered with Route53 in your account, or that your hosted zone has the appropriate NS records.
To transfer domain registration in Route53, see:
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-transfer-to-route-53.html
To update the NS records in your hosted zone, see:
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/SOA-NSrecords.html#NSrecords
```

I tried the links in the error message, but they are the same steps I followed to create the hosted zone in the first place. Any help to point me in the right direction would be much appreciated.

Thanks!

**Comment by huanjani** [comment]
Hi, @sethtjf! 
Hmmm.... When you go to the Route 53 console in your AWS account, do you see the domain in question under "registered domains"?

**Comment by h5aaimtron** [comment]
@huanjani I'm having the same issue. As the original poster mentions, it is a domain that was not registered through Route53 so it would not show up under registered domains. That would be similar to my issue. I have delegated responsibility of a subdomain from one account to another account. All queries against the subdomain look good and records for static s3 sites are working through it, however; copilot tells me I don't own the subdomain when running against the account with the subdomain hosted zone.

I might add that it prevents me from using the alias property when deploying services which is a big issue.

**Comment by efekarakus** [comment]
We created a PR https://github.com/aws/copilot-cli/pull/4281 to display the wording only if the NS records are not setup properly so hopefully this issue should go away now.

And @h5aaimtron I think got to the bottom of his concern in https://github.com/aws/copilot-cli/issues/4270#issuecomment-1353492786

**Comment by Lou1415926** [comment]
Hello all! The fix mentioned by @efekarakus  https://github.com/aws/copilot-cli/issues/4172#issuecomment-1357979545 is now released in [v1.25.0](https://github.com/aws/copilot-cli/releases/tag/v1.25.0)!
